doc/client.rst

```diff
@@ -993,6 +993,21 @@ Index Operations
             client.index_geo2dsphere_create('test', 'pads', 'loc', 'pads_loc_geo')
             client.close()
 
+    .. method:: index_cdt_create(ns: str, set: str, bin: str, index_type, index_datatype, index_name: str, ctx: dict[, policy: dict])
```

Comment:
Should `ctx` indicate that the `ctx` key in the dict is required?

Reply:
I already mentioned this in this [line](https://github.com/aerospike/aerospike-client-python/pull/504/files#diff-1f671cf216bc96085c0b90e689aaf6bb7a563ffa14b3dc3108ba6cfcadd6e92fR1008).

Reply:
Yea, just seemed it belongs here too - but I don't know the format here, so feel free to ignore.